Client/Frontend/Strings.swift

```diff
@@ -4,13 +4,14 @@
 
 import Foundation
 
-public struct Strings {}
+public struct Strings {
+    public static let bundle = Bundle(for: BundleClass.self)
+}
 
 class BundleClass {}
 
 func MZLocalizedString(_ key: String, tableName: String? = nil, value: String = "", comment: String) -> String {
-    let bundle = Bundle(for: BundleClass.self)
-    return NSLocalizedString(key, tableName: tableName, bundle: bundle, value: value, comment: comment)
+    return NSLocalizedString(key, tableName: tableName, bundle: Strings.bundle, value: value, comment: comment)
 }
 
```

Comment:
Nit: Is there a reason we are adding `public static let bundle = Bundle(for: BundleClass.self)` to `public struct Strings` and not making another `extension Strings` like below 🤔 


```
public struct Strings {}

class BundleClass {}

func MZLocalizedString(_ key: String, tableName: String? = nil, value: String = "", comment: String) -> String {
    return NSLocalizedString(key, tableName: tableName, bundle: Strings.bundle, value: value, comment: comment)
}

extension Strings {
    public static let bundle = Bundle(for: BundleClass.self)
}

Reply:
I don't think there's any reason either way. Very happy to take guidance from you about the Swiftyness of one approach over the other.

Reply:
Specific to Swiftyness I feel you have good grasp of the domain and can likely teach me a thing or two ha 😅 

Honestly, they both are fine and I'd suggest the approach I mentioned only to be consistent with `Strings` file but if you are writing basic code in general then your approach is also good 👌🏼

Reply:
For now no changes are required for this comment 🙂